src/display/display_utils.js

```diff
@@ -23,6 +23,19 @@ import { BaseException, stringToBytes, Util, warn } from "../shared/util.js";
 
 const SVG_NS = "http://www.w3.org/2000/svg";
 
+const IdRandomPrefix = (() => {
+  if (typeof globalThis.crypto !== "undefined") {
+    return `${globalThis.crypto.randomUUID()}-`;
+  }
```

Comment:
What's the performance of this code like?

---

Given that we use a hard-coded prefix string with the Editing-functionality, I suppose that we *could try* a similar approach here as well if that'd make things simpler and more efficient. (As long as the resulting `id` is *unlikely* to clash with named destinations.)

Reply:
I ran this on my powerful computer:
```
s = performance.now(); for (let i = 0; i < 1e7; i++) { crypto.randomUUID(); }; d = performance.now() - s 
```

and I got `9475ms` so a unique call should be around `1ms`, so in considering that it's called one time I think it's acceptable.
That said, it's on my powerful computer...

I use the ids in some tests (here and in m-c (i.e. https://searchfox.org/mozilla-central/search?q=%23pdfjs_internal_editor_&path=&case=false&regexp=false)). 
I think we could use `isInAutomation` and/or preprocessing stuff we have here to generate a specific id for test or add an attribute... I'm not sure that's a good idea...